homework9/homework9/Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+
+namespace homework9
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите число n:");
+            int n = int.Parse(Console.ReadLine());
+
+            var myArray = new double[n];
+
+            Random rnd = new Random();
+            for (int i = 0; i < (n-1); i++) 
```

Comment:
Почему последний элемент массива не заподняется?
```suggestion
            for (int i = 0; i < n; i++) 
```